### PR TITLE
Adding FileBug Icon

### PR DESCRIPTION
**Pull request by cainhimself**
*Describe the change you are making*

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

**Comment by cainhimself** [comment]
@unindented , @chrismohr  here's the error file icon added.
Please review.

File: src/components/Illustration/illustrations/PraiseGiftSmaller72.tsx [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-        <circle cx="-35.231" cy="36.753" r={36} fill="#386cbb" transform="rotate(-88.789)" />
-  <path fill="#ffc56c" d="M40.653 23.425l-5.136-10.032c-1.2-2.352-4.368-2.736-6.144-.816a3.76 3.76 0 0 0 1.152 5.904z" />
-  <path fill="#ffc56c" d="M32.445 23.425l5.136-10.032c1.2-2.352 4.368-2.736 6.144-.816a3.76 3.76 0 0 1-1.152 5.904z" />
-  <path fill="#fb7f78" d="M49.245 58.08H23.758c-3.264 0-5.904-2.592-6.048-5.808l-1.008-26.015a6.02 6.02 0 0 1 6.048-6.288H50.3c3.456 0 6.192 2.88 6.048 6.288l-1.008 26.015c-.144 3.264-2.832 5.808-6.096 5.808z" />
-  <path fill="#ffc56c" d="M55.677 41.76h-38.4l-.24-6.575h38.976z" />
-  <path fill="#ffc56c" d="M33.31 19.873h6.527V57.84H33.31z" />
-      </svg>
-    );
-  }
-}
```
Comment (unindented) [inline src/components/Illustration/illustrations/PraiseGiftSmaller72.tsx]: Ah, it looks like this file wasn't in the `assets/` folder, so it got removed. @Logrologist can you confirm it's not needed?

**Comment by smoak** [comment]
@swese44 this is the PR you helped me with. lgtu?

File: src/components/Illustration/illustrations/PraiseGiftSmaller72.tsx (hunk 1 above)
Comment (swese44) [reply src/components/Illustration/illustrations/PraiseGiftSmaller72.tsx]: it looks like this file was generated then the asset renamed all in the original PR, so it's correct that it should be removed

**Comment by swese44** [comment]
looks like a different diff image was affected, i'll re-run them now and push the changes up

**Pull request merged**